.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,50 +1,17 @@
-<!--- Use this part of template if you're adding new package. Remove the rest. -->
```

Comment:
This file should not be touched.

Please check update instructions:
- https://docs.hunter.sh/en/latest/creating-new/update.html